future-apps/actor/contracts/Actor.sol

```diff
@@ -0,0 +1,81 @@
+/*
+ * SPDX-License-Identitifer:    GPL-3.0-or-later
+ */
+
+pragma solidity 0.4.24;
+
+import "@aragon/apps-vault/contracts/Vault.sol";
+
+import "@aragon/os/contracts/common/IForwarder.sol";
+
+
+contract Actor is Vault, IForwarder {
+    bytes32 public constant EXECUTE_ROLE = keccak256("EXECUTE_ROLE");
+    bytes32 public constant RUN_SCRIPT_ROLE = keccak256("RUN_SCRIPT_ROLE");
+
+    string private constant ERROR_EXECUTE_ETH_NO_DATA = "VAULT_EXECUTE_ETH_NO_DATA";
+    string private constant ERROR_EXECUTE_TARGET_NOT_CONTRACT = "VAULT_EXECUTE_TARGET_NOT_CONTRACT";
+
+    event Execute(address indexed sender, address indexed target, uint256 ethValue, bytes data);
+
+    // TODO: requires the @decodeData helper to be implemented in radspec
+    /**
+    * @notice Execute `@decodeData(target, data)` on `target` `ethValue == 0 ? '' : '(Sending' + @formatToken(ethValue, ETH) + ')'`.
```

Comment:
Radspec string won't evaluate until this helper is implemented in radspec or we decide on an alternative

Reply:
What is `decodeData` supposed to do?

Reply:
See https://github.com/aragon/radspec/pull/55